now.json

```diff
@@ -1,10 +1,66 @@
 {
-    "name": "magento-venia",
-    "alias": ["magento-venia", "veniapwa.com"],
-    "engines": {
-        "node": "10.x.x"
+    "version": 2,
+    "name": "venia",
+    "alias": [
+        "magento-venia-develop"
+    ],
+    "scope": "magento-research",
+    "env": {
```

Comment:
Need to add `BRAINTREE_TOKEN: "sandbox_8yrzsvtm_s2bg8fs563crhqzk"` to this list so payments will work

Reply:
It appears this is actually needed at build time, that's why copying doesn't solve the problem. It would make sense the bundle can't expect to have access to `process.env`, so the value has to be built in. Add to the build `env`, and that should resolve.

Reply:
I wasn't copying the .env file!